transitgateway.tf

```diff
@@ -66,15 +66,15 @@ resource "aws_ram_principal_association" "tgw" {
 resource "aws_ec2_transit_gateway_route_table" "tgw_attachments" {
   provider = aws.sharedservicesprovisionaccount
 
-  for_each = merge(local.env_accounts_same_type, local.pca_account_same_type)
+  for_each = merge(local.env_accounts_same_type, local.pca_account_same_type, local.userservices_account_same_type)
```

Comment:
No! We want User Services to use the default TGW route table just like Shared Services does. As as add assessment environments, VPN tunnels, etc. new routes for these things will automatically propagate to the default route table. This is precisely what we want for the Shared Services and User Services.

Reply:
Roger- see https://github.com/cisagov/cool-sharedservices-networking/pull/37/commits/3bc6a4d916fdce8cee81288453cc3ed54f071ea3.